docs/usage/agent-identity.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+parent: Usage
+---
+
+# Agent identity
+
+Zincati agent tries to derive a unique identity for the machine it is running on by introspecting the underlying OS and reading user configuration.
+This includes assigning an ID and a group label specific to the agent, so that cluster-wide upgrades can be orchestrated via [phased rollouts][phased] and [lock-based][fleetlock-strategy] reboots.
+
+[phased]: auto-updates.md#phased-rollouts-client-wariness-canaries
+[fleetlock-strategy]:  updates-strategy.md#lock-based-strategy
+
+## Identity configuration
+
+All agent identity values are normally auto-detected at startup and do not require user intervention.
+
+However, the following settings can be overridden through configuration fragments in the `identity` section:
+ * `group`: group label, used for graph fetching ([Cincinnati][cincinnati]) and reboot orchestration ([FleetLock][fleetlock])
+ * `node_uuid`: agent ID, used for graph fetching ([Cincinnati][cincinnati]) and reboot orchestration ([FleetLock][fleetlock])
+ * `rollout_wariness`: agent wariness to [phased rollouts][phased], used for graph fetching ([Cincinnati][cincinnati]).
+
+The following are defaults for each setting:
+- `group` (group label) is set to `default`
```

Comment:
Any reason not to use the empty string instead?  (A user interface could render it as "(default)") or so?

Reply:
The empty string is not a valid value. If unset in user configuration, the literal `default` value is sent to the FleetLock backend.

(yes, we could have made that field optional and let the backend auto-bucket clients without an explicit setting, when we defined the protocol. But we didn't foresee strong usecases for that, and now it's a bit late)